### PR TITLE
[RFR] Fix links to ra-tree documentation

### LINKED ISSUE #4017: ra-tree-ui read.me 404
Hi, following  https://github.com/marmelab/react-admin/tree/master/packages/ra-tree-core. leads 404 page for some reason. I am wondering if this is under rebuilding phase?

### PR DESCRIPTION
**Pull request by fzaninotto**
Closes #4017

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ See the [Data Provider](./DataProviders.md#available-providers) page.
...
-- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/): Components providing the logic but no UI to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
+- [marmelab/ra-realtime](https://github.com/marmelab/react-admin/tree/2.x/packages/ra-realtime): enable realtime updates
+- [marmelab/ra-tree-ui-materialui](https://github.com/marmelab/react-admin/tree/2.x/packages/ra-tree-ui-materialui): Components to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
```
Comment (djhi) [inline docs/Ecosystem.md]: same

File: docs/Ecosystem.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ See the [Data Provider](./DataProviders.md#available-providers) page.
...
+- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/2.x/packages/ra-tree-core/): Components providing the logic but no UI to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
```
Comment (djhi) [inline docs/Ecosystem.md]: same

File: docs/Ecosystem.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ See the [Data Provider](./DataProviders.md#available-providers) page.
 
 ## Miscellaneous
 
-- [marmelab/ra-realtime](https://github.com/marmelab/react-admin/tree/master/packages/ra-realtime): enable realtime updates
-- [marmelab/ra-tree-ui-materialui](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/): Components to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
-- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/): Components providing the logic but no UI to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
+- [marmelab/ra-realtime](https://github.com/marmelab/react-admin/tree/2.x/packages/ra-realtime): enable realtime updates
```
Comment (djhi) [inline docs/Ecosystem.md]: Shouldn't we add a note indicating its for v2 only ?

File: docs/Labs.md [hunk 4]
```diff
@@ -13,5 +13,5 @@ This section references some projects we're working which are not ready yet for
 
 `ra-tree` is a set of components allowing to work with tree data. It is composed of two packages:
 
-- `ra-tree-core`: The core components which only provides logic and no UI. Read more about it in its [README](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/README.md)
-- `ra-tree-ui-materialui`: Some UI components using the previous package and built with material-ui. Read more about it in its [README](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/README.md)
+- `ra-tree-core`: The core components which only provides logic and no UI. Read more about it in its [README](https://github.com/marmelab/react-admin/blob/2.x/packages/ra-tree-core/README.md)
```
Comment (djhi) [inline docs/Labs.md]: same

File: docs/Labs.md [hunk 5]
```diff
@@ -13,5 +13,5 @@ This section references some projects we're working which are not ready yet for
...
+- `ra-tree-ui-materialui`: Some UI components using the previous package and built with material-ui. Read more about it in its [README](https://github.com/marmelab/react-admin/blob/2.x/packages/ra-tree-ui-materialui/README.md)
```
Comment (djhi) [inline docs/Labs.md]: same

File: docs/List.md [hunk 6]
```diff
@@ -1221,7 +1221,7 @@ export const CategoriesList = (props) => (
 );
 ```
 
-To learn more about this component features, please refers to its [README](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/README.md).
+To learn more about this component features, please refers to its [README](https://github.com/marmelab/react-admin/blob/2.x/packages/ra-tree-ui-materialui/README.md).
```
Comment (djhi) [inline docs/List.md]: same

File: docs/Ecosystem.md (hunk 3 above)
Comment (djhi) [reply docs/Ecosystem.md]: sorry, this comment should have come first

Comment (fzaninotto) [reply docs/Ecosystem.md]: it's the v2 documentation. If we start speaking about v3 in the v2 docs, we're in for a long and hard work!

Comment (djhi) [reply docs/Ecosystem.md]: Ah sorry I missed the target branch :ok_hand:

**Pull request merged**